packages/flame_test/example/pubspec.yaml

```diff
@@ -16,7 +16,7 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  flame_test: ^1.0.0
+  flame_test: ^1.1.0
   test: ^1.17.10
```

Comment:
This is incompatible with latest `test` version 1.20.1

Reply:
But we are not depending on the latest version, so it should be able to resolve an older version, or what do you mean?

Reply:
Nvm, seems like it's `flutter_test`'s problem: they depend on a very specific version of `test_api`, which makes them incompatible with the latest version of `test`.